admin/app/controllers/admin/OAuthLoginAdminController.scala

```diff
@@ -20,7 +21,14 @@ class OAuthLoginAdminController(
       Ok(views.html.auth.login(error))
     }
   override def googleAuthConfig(request: Request[AnyContent]): Option[GoogleAuthConfig] = {
-    val host = Some(s"${if (request.secure) "https" else "http"}://${request.host}")
-    conf.GoogleAuth(host).config
+    val currentHost = Some(s"${if (request.secure) "https" else "http"}://${request.host}")
```

Comment:
Another one for the future - it's pretty simple to provide a setup script that enables a secure local endpoint using[ dev-nginx](https://github.com/guardian/dev-nginx). `www.local.dev-theguardian.com` `preview.local.dev-gutools.co.uk` etc.

Means we could get rid of all this `request.secure` business presumably and would mirror our other environments more closely.

Reply:
Thanks for the suggestion. Sounds very reasonable. I have created an issue to do this: https://github.com/guardian/frontend/issues/24814